internal/util/update/resource-merge.go

```diff
@@ -103,23 +104,31 @@ func (u ResourceMergeUpdater) updatedKptfile(updatedPath string, options UpdateO
 		return kptfile.KptFile{}, err
 	}
 	commit := strings.TrimSpace(string(b))
-	kf, err := kptfileutil.ReadFile(updatedPath)
+	updatedKf, err := kptfileutil.ReadFile(updatedPath)
+	if err != nil {
+		updatedKf, err = kptfileutil.ReadFile(options.PackagePath)
```

Comment:
Is this right to read both updatedK and originalKf from the same file?

Reply:
This means we are using local in case there is no file in updated. If user deletes Kptfile from upstream, we should retain local Kptfile in my opinion.